### ISSUE TITLE
comma-style: breaks for object properties having line breaks

### ISSUE THREAD

**Issue by vegetableman** [eslint/eslint#1314]
The following code throws an error `',' should be placed first.` under default setting of `last` option:

``` javascript
var obj = {
    "a": 1,
    "b": 2, "c":
    3
};
```

which is a valid case in jshint.
